src/templates/preview.js

```diff
@@ -126,6 +127,11 @@ const JournalPage = ({
   let blogs = postGroup
   return(
   <Layout>
+    <SEO 
+      title="The Entrepreneur's Blog | Codebrahma"
+      description="Codebrahma, premium engineering company specializing in web and mobile applications. Blog is where we talk about products, growth and everything startup."
+      keywords={[`ruby on rails`, `Ruby on Rails development`, `angularjs framework`, `angularjs code`, `meteor js`, `react js`, `reactjs native`, `node js development`, `node js website`, `spree commerce`, `ruby on rails developer`, `react js components`, `javascript meteor`, `Angular javascript`, `angular framework`, `js angular`, `angular website`, `Android Website`, `ios app development`, `mobile web app`, `javascript android app`, `web application development company`]}
+    />
```

Comment:
This is incorrect. Why adding SEO for `Preview` template? 
We have to add SEO only for pages. Not for templates.